.happy/terraform/modules/batch/main.tf

```diff
@@ -30,7 +30,7 @@ resource aws_batch_job_definition batch_job_def {
     },
     {
       "name": "REMOTE_DEV_PREFIX",
-      "value": "/${var.custom_stack_name}"
+      "value": "${var.remote_dev_prefix}"
```

Comment:
This is used as db schema prefix also right? Not sure if any python code needs changes as a result of this?

Reply:
Nope - we just add the leading '/' earlier on in the workflow here:
https://github.com/chanzuckerberg/corpora-data-portal/pull/902/files#diff-667e21ce425cdbf70b0b62e040edb6a4ec8c1100a20dd94fc36cda064f98e100R13